presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/AddExchanges.java

```diff
@@ -1397,5 +1406,10 @@ private boolean isCandidate(Map<ColumnHandle, NullableValue> bindings)
 
             return true;
         }
+
+        private <T> boolean intersects(Set<T> one, Set<T> other)
```

Comment:
`one, other` -- this doesn't sound great.
`first, second` ? `a, b` ?

Reply:
I think I used to use `first` and `second`... but I got comments like I should use `other` instead. I will use `first` and `second`